L1Trigger/TrackTrigger/plugins/TTStubBuilder.cc

```diff
@@ -274,9 +314,8 @@ void TTStubBuilder<Ref_Phase2TrackerDigi_>::produce(edm::Event& iEvent, const ed
 
               if (CIC_reject)  // The stub added does not pass the cut
               {
-                tempTTStub2.setTriggerDisplacement(1000 + 2. * tempTTStub.getTriggerDisplacement());
-                tempTTStub2.setTriggerOffset(1000 + 2. * tempTTStub.getTriggerOffset());
-                tempTTStub2.setRealTriggerOffset(1000 + 2. * tempTTStub.getRealTriggerOffset());
+                tempTTStub2.setRawBend(1000 + 2. * tempTTStub.getRawBend());
```

Comment:
magic number 1000 should be named constant

Reply:
fixed